Sake!

fludd12 opened the pull request:
:cl: Fel and LeonDuvall
add: You can now drink sake! Can be found in the bar, or made with rice
and sugar.
/:cl:

Sprites and idea courtesy of LeonDuvall.
Can be made with 15u rice and 5u sugar at 400k.

fludd12 committed:
Merge remote-tracking branch 'refs/remotes/Citadel-Station-13/master'

fludd12 committed:
Merge remote-tracking branch 'refs/remotes/Citadel-Station-13/master'

fludd12 committed:
Sake!

:cl: Fel and LeonDuvall
add: You can now drink sake! Can be found in the bar, or made with rice
and sugar.
/:cl:

Sprites and idea courtesy of LeonDuvall.
Can be made with 15u rice and 5u sugar at 400k.

deathride58 commented on code/game/machinery/vending.dm:
```diff
@@ -640,7 +640,7 @@ IF YOU MODIFY THE PRODUCTS LIST OF A MACHINE, MAKE SURE TO UPDATE ITS RESUPPLY C
 					/obj/item/reagent_containers/food/drinks/drinkingglass = 30, /obj/item/reagent_containers/food/drinks/ice = 10,
 					/obj/item/reagent_containers/food/drinks/drinkingglass/shotglass = 12, /obj/item/reagent_containers/food/drinks/flask = 3,
-					/obj/item/reagent_containers/food/drinks/beer = 6)
+					/obj/item/reagent_containers/food/drinks/beer = 6, /obj/item/reagent_containers/food/drinks/bottle/sake = 5)
```
add `//CIT CHANGE - adds sake to boozeomat` here, or modularize the change

fludd12 replied:
Done!

fludd12 committed:
Does the thing

Modularization of vending machines would require a lot more effort than
is remotely worth for such a simple change. Fortunately, not a lot of
new alcohols are added to it often, so it should be fine.
